Library/Homebrew/cmd/audit.rb

```diff
@@ -659,6 +659,27 @@ def audit_line(line, lineno)
     end
   end
 
+  def audit_prefix_has_contents
+    return unless formula.prefix.directory?
+
+    def installation_file? f
+      if File.directory?(f)
+        false
+      else
+        f = File.basename(f)
+        !(Metafiles.copy?(f) || %w[.DS_Store INSTALL_RECEIPT.json].include?(f))
+      end
+    end
+
+    unless Dir.glob("#{formula.prefix}/**/*").any? { |f| installation_file? f }
+      problem <<-EOS.undent
+        The installation seems to be empty. Please ensure the prefix
+        is set correctly and expected files are installed.
+        The prefix configure/make argument may be case-sensitive.
+      EOS
+    end
```

Comment:
I guess it refers to [this block](https://github.com/Homebrew/homebrew/blob/master/Library/Homebrew/metafiles.rb#L17-L18) of Metafiles code.

Everything that is in the (prior version) of ` osh `'s opt directory:

```
AUTHORS
CHANGES
INSTALL_RECEIPT.json
LICENSE
NOTES
README
```